src/web/app/pages-instructor/instructor-session-edit-page/__snapshots__/instructor-session-edit-page.component.spec.ts.snap

```diff
@@ -4039,7 +4039,7 @@ exports[`InstructorSessionEditPageComponent should snap with new question added
                   <div
                     class="offset-1 col-md-4 text-md-right"
                   >
-                     The maximum number of students each respondent should give feedback to: 
+                     The maximum number of students_excluding_self each respondent should give feedback to: 
```

Comment:
This unfortunately is no longer user-friendly. If previously we can just lowercase (I assume), we may either need a dedicated pipe here, or maybe just trimming the `_EXCLUDING_SELF` at the end.